src/ops.rs

```diff
@@ -238,19 +244,19 @@ fn serialize_response(
 }
 
 #[inline]
-fn ok_future(buf: Buf) -> Box<Op> {
+pub fn ok_future(buf: Buf) -> Box<Op> {
   Box::new(futures::future::ok(buf))
 }
 
 // Shout out to Earl Sweatshirt.
 #[inline]
-fn odd_future(err: DenoError) -> Box<Op> {
+pub fn odd_future(err: DenoError) -> Box<Op> {
   Box::new(futures::future::err(err))
 }
 
 // https://github.com/denoland/deno/blob/golang/os.go#L100-L154
 fn op_code_fetch(
-  state: &IsolateState,
+  state: &Arc<IsolateState>,
```

Comment:
I wonder if we should refactor so that IsolateState itself has Send, instead of wrapping in an Arc everywhere. After all the Mutex is already internalized...

Reply:
Sure. These types of refactors are generally easy to do due to the type system. I'd leave this as is for now because of laziness...